backend/service/authn/sql.go

```diff
@@ -0,0 +1,53 @@
+package authn
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/lyft/clutch/backend/service"
+	"github.com/lyft/clutch/backend/service/db/postgres"
+)
+
+type authnUser struct {
+	id                   string
+	providerRefreshToken []byte
+}
+
+type repository struct {
+	db *sql.DB
+}
+
+func newRepository() (*repository, error) {
+	svc, ok := service.Registry[postgres.Name]
+	if !ok {
+		return nil, fmt.Errorf("no database registered")
```

Comment:
are we planning to support other databases in the future? If yes, it might be helpful to display the name of the db in the error message i.e fmt.Errorf("%s database not registered", postgres.Name)?